### PR TITLE
feat(cli): add cli install links

### PR DESCRIPTION
**Pull request by nimbinatus**
fixes https://github.com/pulumi/pulumi-hugo/issues/1672

### CODE REVIEW

File: themes/default/content/learn/abstraction-encapsulation/_index.md [hunk 1]
```diff
@@ -32,7 +32,10 @@ You will need the following knowledge to complete this pathway:
 
 Optionally, if you want to build some of the examples yourself to experiment, you'll need the following tools and skills:
 
-- A [Pulumi account and token]({{< ref "/docs/intro/pulumi-service/accounts#access-tokens" >}}) (If you don't have an account, go to the [signup page](https://app.pulumi.com/signup))
+- A [Pulumi account and token]({{< ref "/docs/intro/pulumi-service/accounts#access-tokens" >}})
+    - If you don't have an account, go to the[signup page](https://app.pulumi.com/signup).
+- The [Pulumi CLI]({{< ref "/docs/reference/cli" >}})
+    - If you don't have the CLI, go to the [installation page]({{< ref "/docs/get-started/install" >}})
```
Comment (cnunciato) [inline themes/default/content/learn/abstraction-encapsulation/_index.md:38]:
```suggestion
- A [Pulumi account and token]({{< ref "/docs/intro/pulumi-service/accounts#access-tokens" >}})
    - If you don't have an account, go to the [signup page](https://app.pulumi.com/signup).
- The [Pulumi CLI]({{< ref "/docs/reference/cli" >}})
    - If you don't have the CLI, go to the [installation page]({{< ref "/docs/get-started/install" >}}).
```

File: themes/default/content/learn/building-with-pulumi/_index.md [hunk 2]
```diff
@@ -33,6 +33,9 @@ pathway]({{< relref "learn/pulumi-fundamentals" >}}) already.
 You will need the following tools to complete this module:
 
 * A [Pulumi account and token]({{< ref "/docs/intro/pulumi-service/accounts#access-tokens" >}})
+    * If you don't have an account, go to the[signup page](https://app.pulumi.com/signup).
+* [Pulumi CLI]({{< ref "/docs/reference/cli" >}})
+    * If you don't have the CLI, go to the [installation page]({{< ref "/docs/get-started/install" >}})
```
Comment (cnunciato) [inline themes/default/content/learn/building-with-pulumi/_index.md:38]:
```suggestion
    * If you don't have an account, go to the [signup page](https://app.pulumi.com/signup).
* [Pulumi CLI]({{< ref "/docs/reference/cli" >}})
    * If you don't have the CLI, go to the [installation page]({{< ref "/docs/get-started/install" >}}).
```

File: themes/default/content/learn/embedding-pulumi/_index.md [hunk 3]
```diff
@@ -33,7 +33,9 @@ How long this tutorial will take depends on your internet connection, reading sp
 You will need the following tools to complete this pathway:
 
 - A [Pulumi account and token]({{< relref "/docs/intro/pulumi-service/accounts#access-tokens" >}})
-  - If you don't have an account, go to the [signup page](https://app.pulumi.com/signup).
+    - If you don't have an account, go to the [signup page](https://app.pulumi.com/signup).
+- The [Pulumi CLI]({{< ref "/docs/reference/cli" >}})
+    - If you don't have the CLI, go to the [installation page]({{< ref "/docs/get-started/install" >}})
```
Comment (cnunciato) [inline themes/default/content/learn/embedding-pulumi/_index.md:38]:
```suggestion
    - If you don't have an account, go to the [signup page](https://app.pulumi.com/signup).
- The [Pulumi CLI]({{< ref "/docs/reference/cli" >}})
    - If you don't have the CLI, go to the [installation page]({{< ref "/docs/get-started/install" >}}).
```

File: themes/default/content/learn/importing/_index.md [hunk 4]
```diff
@@ -28,7 +28,9 @@ How long this tutorial will take depends on your internet connection, reading sp
 You will need the following tools to complete this pathway:
 
 - A [Pulumi account and token]({{< relref "/docs/intro/pulumi-service/accounts#access-tokens" >}})
-  - If you don't have an account, go to the [signup page](https://app.pulumi.com/signup).
+    - If you don't have an account, go to the [signup page](https://app.pulumi.com/signup).
+- The [Pulumi CLI]({{< ref "/docs/reference/cli" >}})
+    - If you don't have the CLI, go to the [installation page]({{< ref "/docs/get-started/install" >}})
```
Comment (cnunciato) [inline themes/default/content/learn/importing/_index.md:33]:
```suggestion
    - If you don't have the CLI, go to the [installation page]({{< ref "/docs/get-started/install" >}}).
```

File: themes/default/content/learn/pulumi-fundamentals/_index.md [hunk 5]
```diff
@@ -37,8 +37,9 @@ minutes to complete.
 You will need the following tools to complete this tutorial:
 
 - A [Pulumi account and token]({{< ref "/docs/intro/pulumi-service/accounts#access-tokens" >}})
-  - If you don't have an account, go to the
-    [signup page](https://app.pulumi.com/signup).
+    - If you don't have an account, go to the[signup page](https://app.pulumi.com/signup).
+- The [Pulumi CLI]({{< ref "/docs/reference/cli" >}})
+    - If you don't have the CLI, go to the [installation page]({{< ref "/docs/get-started/install" >}})
```
Comment (cnunciato) [inline themes/default/content/learn/pulumi-fundamentals/_index.md:42]:
```suggestion
    - If you don't have an account, go to the [signup page](https://app.pulumi.com/signup).
- The [Pulumi CLI]({{< ref "/docs/reference/cli" >}})
    - If you don't have the CLI, go to the [installation page]({{< ref "/docs/get-started/install" >}}).
```

**Review by cnunciato — APPROVED** [review]
Little things, but otherwise 👍 . (Although I do think these work well on the same bullet line -- "The Pulumi CLI. If you don't have the CLI, visit the installation page,", etc. Fewer bullets feels a bit less visually noisy to me, but this is technically fine, too.)

**Comment by nimbinatus** [comment]
Thanks, @cnunciato!

**Pull request merged**